src/pages/office/cat-cpc/components/assessment-report/assessment-report.ts

```diff
@@ -0,0 +1,37 @@
+import { Component, Input, Output, EventEmitter, OnChanges } from '@angular/core';
+import { FormGroup, FormControl } from '@angular/forms';
+
+@Component({
+  selector: 'assessment-report',
+  templateUrl: 'assessment-report.html',
+})
+export class AssessmentReportComponent implements OnChanges {
+
+  @Input()
+  outcome: string;
```

Comment:
is the outcome used in the TS or HTML? if not please remove

Reply:
i was using it to determine if the card was to be displayed prior to moving the logic to the office ts removed.